data_safe_haven/administration/users/guacamole_users.py

```diff
@@ -90,43 +55,8 @@ def list(self) -> Sequence[ResearchUser]:
 
     def remove(self, users: Sequence[ResearchUser]) -> None:
         """Remove list of users from Guacamole"""
```

Comment:
```suggestion
```

Reply:
Sorry - I have implementations for these stubs but they aren't quite working atm.